app/controllers/api/v1/logics_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::LogicsController < Api::V1::Base
+  inherit_resources
+  defaults finder: :find_by_slug!
+
+  actions :show
+  respond_to :xml, :rdf, only: %i(show)
+
+  def show
+    super do |format|
+      format.xml do
+        render :show, content_type: 'application/rdf+xml'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.